pkg/operator/reports_test.go

```diff
@@ -92,3 +99,224 @@ func TestGetNextReportPeriod(t *testing.T) {
 		})
 	}
 }
+
+func TestIsReportRunning(t *testing.T) {
+	const (
+		testNamespace  = "default"
+		testReportName = "test-report"
+		testQueryName  = "test-query"
+	)
+
+	schedule := &metering.ReportSchedule{
+		Period: metering.ReportPeriodCron,
+		Cron:   &metering.ReportScheduleCron{Expression: "5 4 * * *"},
+	}
+
+	reportStart := &time.Time{}
+	reportEndTmp := reportStart.AddDate(0, 1, 0)
+	reportEnd := &reportEndTmp
+
+	testTable := []struct {
+		name       string
+		report     *metering.Report
+		expectFlag bool
+	}{
+		{name: "new report returns false", report: testhelpers.NewReport(testReportName, testNamespace, testQueryName, reportStart, reportEnd, v1alpha1.ReportStatus{}, nil, false), expectFlag: false},
```

Comment:
Can you put each field onto it's own line so these lines aren't so long? And put `name` onto a separate line from the open `{`